### PR TITLE
lovely-icons updates

### PR DESCRIPTION
**Pull request by douglasdeodato**
lovely-icons updates

looks like my PR was not merged by @jsdelivrbot  : https://github.com/jsdelivr/jsdelivr/pull/14032

I did a new released and did not update, am I doing something wrong? thanks.

### CODE REVIEW

**Comment by douglasdeodato** [comment]
@megawac  thanks for the help, but could you give a ping about it? 

Cheers man.

**Comment by douglasdeodato** [comment]
@megawac  my project has 3 releases, why is not updating auto here? https://github.com/douglasdeodato/lovely-icons/releases

cheers man.

**Comment by megawac** [comment]
@douglasdeodato it only syncs the latest one on the initial sync. Not sure why it hasn't picked up 0.1.2

/cc @MartinKolarik

File: files/lovely-icons/update.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
   "name": "lovely-icons",
   "repo": "douglasdeodato/lovely-icons",
   "files": {
-    "include": ["./css/lovely-icons.min.css", "./lovely-icons/*"]
+    "include": ["./css/lovely-icons.min.css", "./lovely-icons/lovely-icons.min.css", "./lovely-icons/*"]
```
Comment (megawac) [inline files/lovely-icons/update.json]: does `./css/lovely-icons` exist? https://github.com/douglasdeodato/lovely-icons I don't see it

Comment (douglasdeodato) [inline files/lovely-icons/update.json]: not anymore, I removed this folder, but the other 2 ones exists.

Comment (MartinKolarik) [inline files/lovely-icons/update.json]: That's probably why the bot hasn't picked up 0.1.2. If it doesn't exist anymore, you should remove it from update.json.

Comment (douglasdeodato) [inline files/lovely-icons/update.json]: @MartinKolarik  and @megawac  I just put back the folder and release a new version, so now the bot should pick this version? thanks for the help.

**Pull request merged**